pages/desktop/details.py

```diff
@@ -69,11 +69,6 @@ class Details(Base):
     _reviews_locator = (By.CSS_SELECTOR, "section#reviews div")
     _add_review_link_locator = (By.ID, "add-review")
 
-    _add_to_collection_locator = (By.CSS_SELECTOR, ".collection-add.widget.collection")
```

Comment:
This is referenced in `click_add_to_collection_widget`, which should also be removed.